go-controller/pkg/address_set/address_set.go

```diff
@@ -1,4 +1,4 @@
-package ovn
+package address_set
```

Comment:
Extreme bikeshedding: address_set is only used by the master code - so maybe it should be in `pkg/ovn/address_set`?

Reply:
moved